voice/transfer-call-inline-ncco.py

```diff
@@ -8,30 +8,21 @@
 envpath = join(dirname(__file__), './.env')
 load_dotenv(envpath)
 
-# Init the client
-print(os.getenv("NEXMO_PRIVATE_KEY"))
+# Init the nexmo client
 client = nexmo.Client(
     application_id=os.getenv('NEXMO_APPLICATION_ID'),
     private_key=os.getenv("NEXMO_PRIVATE_KEY")
 )
 
-response = client.create_call({
-    "to": [{"type": "phone", "number": os.getenv('TO_NUMBER')}],
-    "from": {"type": "phone", "number": os.getenv('FROM_NUMBER')},
-    "ncco": [
-        {
-            "action": "talk",
-            "text": "This is just a text whilst you tranfer to another NCCO"
-        }
-    ]
-})
+uuid = os.getenv('CALL_UUID')
```

Comment:
This should be `UUID`, per the [specs](https://github.com/Nexmo/code-snippet-specs/blob/master/voice/transfer-a-call-with-ncco.md#variables)